Packs/Rapid7_InsightIDR/Integrations/Rapid7_InsightIDR/insightidr.yml

```diff
@@ -0,0 +1,630 @@
+    - default: false
+      description: The number of log entries to return per page. Default of 50
+      isArray: false
+      name: logs_per_page
+      required: false
+      secret: false
+    - default: false
+      description: the earlier sequence number of a log entry to start searching from
+      isArray: false
+      name: sequence_number
+      required: false
+      secret: false
+    deprecated: false
+    description: Query inside a log for certain values.
+    execution: false
+    name: rapid7-insight-idr-query-log
+    outputs:
+    - contextPath: Rapid7InsightIDR.Event.log_id
+      description: Event message
+      type: String
+    - contextPath: Rapid7InsightIDR.Event.message
+      description: ID of the log the event appears in
+      type: String
+    - contextPath: Rapid7InsightIDR.Event.timestamp
+      description: Time when the event fired
+      type: Number
+  - arguments:
+    - default: false
+      description: log set ID
+      isArray: true
+      name: log_set_id
+      required: true
+      secret: false
+    - default: false
+      description: "A valid LEQL query to run against the logmore info: https://docs.rapid7.com/insightidr/build-a-query/"
+      isArray: false
+      name: query
+      required: true
+      secret: false
+    - default: false
+      description: An optional time range string (ie 1 week, 1 day) - While using
+        this parameter, start_time and end_time isn't needed
+      isArray: false
+      name: time_range
+      required: false
+      secret: false
+    - default: false
+      description: 'Lower bound of the time range you want to query against. Format:
+        UNIX timestamp in milliseconds. Example:1450557004000'
+      isArray: false
+      name: start_time
+      required: false
+      secret: false
+    - default: false
+      description: 'Upper bound of the time range you want to query against. Format:
+        UNIX timestamp in milliseconds. Example:1460557604000'
+      isArray: false
+      name: end_time
+      required: false
+      secret: false
+    - default: false
+      description: The number of log entries to return per page. Default of 50
+      isArray: false
+      name: logs_per_page
+      required: false
+      secret: false
+    - default: false
+      description: the earlier sequence number of a log entry to start searching from
+      isArray: false
+      name: sequence_number
+      required: false
+      secret: false
+    deprecated: false
+    description: Query inside a log set for certain values.
+    execution: false
+    name: rapid7-insight-idr-query-log-set
+    outputs:
+    - contextPath: Rapid7InsightIDR.Event.log_id
+      description: Event message
+      type: String
+    - contextPath: Rapid7InsightIDR.Event.message
+      description: ID of the log the event appears in
+      type: String
+    - contextPath: Rapid7InsightIDR.Event.timestamp
+      description: Time when the event fired
+      type: Number
```

Comment:
is it in UNIX time? if not, than the type is a `Date`

Reply:
yes, it's UNIX timestamp in milliseconds